dist/k8s/prod/frontend.yml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
       - name: frontend
-        image: docker.pkg.github.com/wirvsvirus-gemeinsam-sport/gemeinsam-sport-frontend/frontend:latest
+        image: docker.pkg.github.com/wirvsvirus-gemeinsam-sport/gemeinsam-sport-frontend/frontend:0.0.61
```

Comment:
Why hardcoded Version?
Why not prod or stable?

Reply:
Kubernetes only redeploys if the value changes because it otherwise assumes that nothing changed